setup.py

```diff
@@ -1,10 +1,16 @@
 from setuptools import setup, find_packages
+from os import path
 
 packages = find_packages()
-packages.append("")
+
+this_directory = path.abspath(path.dirname(__file__))
+about = {}
+with open(path.join(this_directory, 'gemd', '__version__.py'), 'r') as f:
+    exec(f.read(), about)
```

Comment:
Because I copied this code from citrine-python

https://github.com/CitrineInformatics/citrine-python/blob/54b9fb8a0658dfe0c98315260d1a7e1f972d2a53/setup.py#L10-L11

Reply:
Fair enough. Let's change it, unless you have some objection. I'll do the same for the SDK.

Reply:
So I think the new version hits your goals, but I'm kinda inclined to just have
```
from gemd.version import __version__
```
even though I know that's not wise.

Reply:
I'm just always wary of an unnecessary `exec`, although it's not the most harmful here, sure. You could use the line you put above. But my personal preference is opening the `__version__` file as plaintext, grabbing only its first line, and regexing for the version string. That is, what you've put in, except I don't think we need to have `__version__` as a variable in the version string file.

But it's not a huge deal either way, so I'll leave the final call for gemd up to you.

Reply:
As weird as it is, as far as I can tell having `__version__` in file is actually best practice.  

https://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package

A regex extraction seems gross to me, except it also seems to be the best choice.  We could also try

https://pypi.org/project/setuptools-scm/

if we want to try going down the auto-semantic version approach.

For today, I'll just make this approach work.